index.html

```diff
@@ -1,13 +1,16 @@
-!<!DOCTYPE html>
 <!DOCTYPE html>
 <html>
   <head>
     <meta charset="utf-8">
     <title>BusMall</title>
-    <link rel="stylesheet" href="style.css"
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-
+    <section class=“img-container”>
```

Comment:
When we were building our tables, did we need to have pre-built tr elements hardcoded into the html from the beginning? Or did we draw them in as necessary, and append them to a common ancestor element? Do you think we need the divs left, right, and center, or do you think we can make do with just this section tag?